keylime/ima/ima.py

```diff
@@ -51,6 +53,50 @@ class RUNTIME_POLICY_GENERATOR:
 }
 
 
+RUNTIME_POLICY_SCHEMA = {
+    "type": "object",
+    "required": ["meta", "release", "digests", "excludes", "keyrings", "ima", "ima-buf", "verification-keys"],
+    "properties": {
+        "meta": {
+            "type": "object",
+            "required": ["version", "generator"],
+            "properties": {
+                "version": {"type": "integer", "minimum": 0},
+                "generator": {"type": "integer", "minimum": 0, "maximum": 3},
```

Comment:
IIRC this 0 .. 3 comes from an enum-a-like, maybe the last value should be used, (RUNTIME_POLICY_GENERATOR.LegacyAllowList)

But the best approach is, if possible, use a real enum:

```python
import enum
class RUNTIME_POLICY_GENERATOR(enum.IntEnum):
    Unknown = 0
    EmptyAllowList = 1
    CompatibleAllowList = 2
    LegacyAllowList = 3
```

So now we can use the Enum API:

```python
...
"generator": {"type": "integer", "minimum": min(list(RUNTIME_POLICY_GENERATOR)), "maximum": max(list(RUNTIME_POLICY_GENERATOR))},
```